Sail/src/Sail/entities/components/AnimationComponent.h

```diff
@@ -72,7 +72,17 @@ class AnimationComponent : public Component<AnimationComponent> {
 	
 	Entity* rightHandEntity;
 	Entity* leftHandEntity;
-
+	void setAnimation(const unsigned int index) {
+		animationIndex = index;
+		if (animationIndex > m_stack->getAnimationCount()) {
```

Comment:
Should probably log something cause we're supposed to know how many animations there are?

Reply:
nonissue, there is an enum in animationChanger which is used.